Source/Web View/WebViewController.swift

```diff
@@ -596,15 +623,16 @@ extension WebViewController {
     public func removeErrorDisplay() {
         errorView?.removeFromSuperview()
         errorView = nil
+        showWebView()
     }
    
     /// Override to customize the feature name that appears in the error display.
-    public func featureNameForError(error: NSError) -> String {
+    public func featureNameForError(error: NSError?) -> String {
```

Comment:
hmm why the change to an optional type? we should only be calling this if we actually have a non-nil `NSError` value